### ISSUE TITLE
Uncaught TypeError: Cannot read property 'levelIsPractice' of undefined

### ISSUE THREAD

**Issue by technosaurus** [codecombat/codecombat#4037]
The page currently just gets stuck at "Loading..."

Initially I had similar errors with google analytics (blocked in my hosts file) but  I unblocked it (temporarily, but that is a separate issue) and https://codecombat.com/play still won't load

It seems to be related to [this commit](https://github.com/codecombat/codecombat/commit/296d2c940d8ecd729d098e45e203e2b1182ff86a) isssued within the last 24 hours, so perhaps it should not have been checked in yet.

The following errors are generated:
```
 Uncaught TypeError: Cannot read property 'levelIsPractice' of undefined
     at CampaignView.module.exports.CampaignView.annotateLevels (<anonymous>:1766:24)
     at CampaignView.module.exports.CampaignView.countLevels (<anonymous>:1808:14)
     at CampaignView.module.exports.CampaignView.getRenderData (<anonymous>:1571:24)
     at CampaignView.module.exports.CocoView.render (https://codecombat.com/javascripts/app.js:9660:38)
     at CampaignView.module.exports.CampaignView.onCampaignsLoaded (<anonymous>:2124:17)
     at CampaignView.<anonymous> (https://codecombat.com/javascripts/vendor.js:4293:18)
     at CampaignView.<anonymous> (https://codecombat.com/javascripts/vendor.js:3013:23)
     at triggerEvents (https://codecombat.com/javascripts/vendor.js:4402:57)
     at CampaignsCollection.trigger (https://codecombat.com/javascripts/vendor.js:4341:19)
     at Object.options.success (https://codecombat.com/javascripts/vendor.js:5057:20)
     at fire (https://codecombat.com/javascripts/vendor.js:182:299)
     at Object.fireWith [as resolveWith] (https://codecombat.com/javascripts/vendor.js:187:198)
     at done (https://codecombat.com/javascripts/vendor.js:509:95)
     at XMLHttpRequest.<anonymous> (https://codecombat.com/javascripts/vendor.js:526:215)

```
And
```
Uncaught TypeError: Cannot read property 'levelIsPractice' of undefined
    at CampaignView.module.exports.CampaignView.annotateLevels (<anonymous>:1766:24)
    at CampaignView.module.exports.CampaignView.countLevels (<anonymous>:1808:14)
    at CampaignView.module.exports.CampaignView.getRenderData (<anonymous>:1571:24)
    at CampaignView.module.exports.CocoView.render (https://codecombat.com/javascripts/app.js:9660:38)
    at CampaignView.module.exports.CampaignView.onLoaded (<anonymous>:1459:10)
    at triggerEvents (https://codecombat.com/javascripts/vendor.js:4399:57)
    at SuperModel.trigger (https://codecombat.com/javascripts/vendor.js:4341:19)
    at SuperModel.module.exports.SuperModel.updateProgress (https://codecombat.com/javascripts/app.js:324:101)
    at https://codecombat.com/javascripts/app.js:285:254
    at https://codecombat.com/javascripts/vendor.js:2904:43

```

**Comment by sderickson** [comment]
This looks like a duplicate of #4033 which should be fixed from [this commit](a57e26cd4c8975c029475828232c2ffeda29b387). Try clearing your cache?

**Comment by technosaurus** [comment]
@sderickson Seems to be working now, thanks.  BTW your link has .../commits/... replaced with .../issues/...

**Comment by sderickson** [comment]
Whoops, fixed. Thanks for pointing that out!
